### PR TITLE
Inventory Detail Save Changes

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
Allows you to press the `Edit` button on a property or tax lot detail page, modify the existing fields, and then save the changes

### CODE REVIEW

**Comment by RDmitchell** [comment]
@axelstudios -- how does this work on the backend? Are the saved changes made into an audit log record, and so then do they show up as a new column in the Detail view?

**Comment by axelstudios** [comment]
Right, a new `propertyauditlog` or `taxlotauditlog` entry is created in the database for every edit and a new column appears for that inventory item.

**Comment by RDmitchell** [comment]
Great, can't wait to test it. Hopefully it can get merged in to develop some time today?

File: seed/views/properties.py [hunk 1]
```diff
@@ -1141,11 +1143,13 @@ def put(self, request, taxlot_pk, cycle_pk):
...
                     new_state = new_taxlot_state_serializer.save()
                     taxlot_view.update_state(
-                        self, new_state, description=changed_fields
+                        new_state, description=changed_fields
                     )
                     result.update(
                         {'state': new_taxlot_state_serializer.validated_data}
                     )
+                    # Removing organization key because it's not JSON-serializable - TODO find better solution
+                    result['state'].pop('organization')
```
Comment (nllong) [inline seed/views/properties.py]: yuck... should fix this sometime.

**Comment by RDmitchell** [comment]
Thanks Nick !

**Comment by RDmitchell** [comment]
@nllong -- should we make a PT or GH issue for the icebox for this?

**Pull request merged**